### PR TITLE
Correct method in example

### PR DESCRIPTION
**Pull request by ZacMarcus**

### CODE REVIEW

File: src/Cake.SqlServer/SqlServerAliases.cs [hunk 1]
```diff
@@ -317,7 +317,7 @@ public static SqlConnection OpenSqlConnection(this ICakeContext context, String
         ///             SetCommandTimeout(60);
         ///             using (var connection = OpenSqlConnection(@"Data Source=(LocalDb)\v12.0;Initial Catalog=MyDatabase"))
         ///             {
-        ///                 ExecuteSqlCommand(connection, "...");
+        ///                 SetSqlCommandTimeout(connection, "...");
```
Comment (jnm2) [inline src/Cake.SqlServer/SqlServerAliases.cs]: Perhaps `"..."` should be replaced as well?

Comment (ZacMarcus) [inline src/Cake.SqlServer/SqlServerAliases.cs]: Sorry, I changed the wrong thing.

Comment (jnm2) [inline src/Cake.SqlServer/SqlServerAliases.cs]: If you like, you are able to reuse this same PR by reopening it, amending your previous commit, and force pushing to the same branch.

Comment (ZacMarcus) [inline src/Cake.SqlServer/SqlServerAliases.cs]: I already created a new branch =)

Comment (jnm2) [inline src/Cake.SqlServer/SqlServerAliases.cs]: Oh I see, you deleted the whole fork. :+1:

**Pull request closed without merging**